src/app.module.ts

```
@@ -1,10 +1,13 @@
 import { Module } from '@nestjs/common';
 import { AppController } from './app.controller';
 import { AppService } from './app.service';
+import { UsersService } from './users/users.service';
```

ESLint

[error] 'UsersService' is defined but never used.

(@typescript-eslint/no-unused-vars)